packages/babel-preset-env/src/index.js

```diff
@@ -203,6 +204,7 @@ export default declare((api, opts) => {
   const targets = getTargets(optionsTargets, {
     ignoreBrowserslistConfig,
     configPath,
+    env: forwardEnv ? api.env() : undefined,
```

Comment:
I think this should be a tri-state option. In my particular use case, I need to be able to forward a specific string, not just `api.env()`, because my `api.env()` is an auto-generated mess.

---

Besides just the `production` or `development` state, I encode into the `babel-loader` `envName` whether this config is being used to process things in `node_modules` or not, and whether I'm targeting current browsers or not. Only that last information is relevant for the `browserslist` preset.